app/src/main/res/layout/home_fragment.xml

```diff
@@ -0,0 +1,181 @@
+                android:textColor="#7F8A99"
+                android:layout_marginTop="5dp"
+                android:textSize="14dp"/>
+        </RelativeLayout>
+        <RelativeLayout
+            android:layout_width="0dp"
+            android:layout_height="95dp"
+            android:layout_row="1"
+            android:layout_columnWeight="1"
+            android:layout_column="1"
+            android:background="@drawable/total_sent_count_bg"
+            android:layout_marginLeft="7dp"
+            android:layout_marginBottom="7dp"
+            android:layout_marginTop="7dp">
+            <TextView
+                android:id="@+id/totalSentCountTxt"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="125"
+                android:textColor="#9B42FF"
+                android:textSize="22dp"
+                android:layout_marginTop="25dp"
+                android:layout_marginLeft="10dp" />
+
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_below="@id/totalSentCountTxt"
+                android:layout_alignStart="@id/totalSentCountTxt"
+                android:text="Total Sent"
+                android:textColor="#7F8A99"
+                android:layout_marginTop="5dp"
+                android:textSize="14dp"/>
+        </RelativeLayout>
+        <RelativeLayout
+            android:layout_width="0dp"
+            android:layout_height="95dp"
+            android:layout_row="2"
+            android:layout_columnWeight="1"
+            android:layout_column="0"
+            android:background="@drawable/completed_count_bg"
+            android:layout_marginTop="7dp"
+            android:layout_marginRight="7dp">
+            <TextView
+                android:id="@+id/completedCountTxt"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="125"
+                android:textColor="#1684A6"
+                android:textSize="22dp"
+                android:layout_marginTop="25dp"
+                android:layout_marginLeft="10dp" />
+
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_below="@id/completedCountTxt"
+                android:layout_alignStart="@id/completedCountTxt"
+                android:text="Completed"
+                android:textColor="#7F8A99"
+                android:layout_marginTop="5dp"
+                android:textSize="14dp"/>
+        </RelativeLayout>
+
+        <RelativeLayout
+            android:layout_width="0dp"
+            android:layout_height="70dp"
+            android:layout_row="2"
+            android:layout_columnWeight="1"
+            android:layout_column="1" />
+    </GridLayout>
+</RelativeLayout>
```

Comment:
my recommendation here and for all the layouts is : always use "sp" for text size and "dp" for views sizes, again Linear Layout could be used

Reply:
Yes you are correct we should use sp, but issue with using sp( Scalable Pixels) is if someone has increased the font size of their mobile from settings, it will increase the font size of our app automatically and it will make our UI distorted.

Reply:
but i'll update it as per your comment

Reply:
i understand your point, but if a user increase font from mobile settings, it must be that he wants to see the texts bigger, so a bigger text in our app would then be desirable.